doc/pipeline.rst

```diff
@@ -161,7 +156,7 @@ reduces memory requirements.
 
 When using :ref:`pdal translate<translate_command>` or
 :ref:`pdal pipeline<pipeline_command>`
-PDAL uses stream mode if possible.  If stream mode can't use used
+PDAL uses stream mode if possible.  If stream mode can't be used
```

Comment:
We should enhance this section to note that one can find out if a stage is streamable or not by looking for the "Streamable Stage" header in any stage document.